features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/addpeople/AddPeopleStateProvider.kt

```diff
@@ -18,30 +18,27 @@ package io.element.android.features.createroom.impl.addpeople
 
 import androidx.compose.ui.tooling.preview.PreviewParameterProvider
 import io.element.android.features.userlist.api.SelectionMode
+import io.element.android.features.userlist.api.UserListState
 import io.element.android.features.userlist.api.aListOfSelectedUsers
 import io.element.android.features.userlist.api.aUserListState
+import io.element.android.libraries.matrix.ui.components.aMatrixUserList
+import kotlinx.collections.immutable.toImmutableList
 
-open class AddPeopleStateProvider : PreviewParameterProvider<AddPeopleState> {
-    override val values: Sequence<AddPeopleState>
+open class AddPeopleStateProvider : PreviewParameterProvider<UserListState> {
```

Comment:
Maybe rename the class and the file?

Reply:
Done, I kept `AddPeople` as a prefix to distinguish the provider from the other one from the userListModule: https://github.com/vector-im/element-x-android/pull/320/commits/57697a3c707f6b3ef3a9cb0c0938d158a660a153